src/app/shared/services/account.service.ts

```diff
@@ -16,6 +16,11 @@ export class AccountService extends BaseBackendService<Account> {
     super(http);
   }
 
+  public getAccount(accountName: string, domainId: string): Observable<Account> {
+    return this.getList({domainid: domainId})
```

Comment:
Please use name parameter in the API call https://cloudstack.apache.org/api/apidocs-4.9/apis/listAccounts.html